scripts/get_famous_people_list/src/get_famous_people_list.py

```diff
@@ -0,0 +1,63 @@
+import requests
+import spacy
+from bs4 import BeautifulSoup
+
+MAIN_WIKI = 'https://en.wikipedia.org/wiki/Lists_of_people_by_nationality'
+BASE_WIKI = 'https://en.wikipedia.org'
+NLP_MODEL = spacy.load('en_core_web_lg')
+
+
+def get_pages(demonym_file):
+    """Gets individual famous people pages by nationality
+    
+    Arguments:
+        demonym_file {txt} -- path to file containing country demonyms
+    
+    Returns:
+        str[] -- list of urls to individual country pages
+    """
+
+    result = requests.get(MAIN_WIKI)
+    soup = BeautifulSoup(result.content, 'html.parser')
+
+    demonyms = []
+    with open(demonym_file) as fp:
+        demonyms = fp.read().splitlines() 
```

Comment:
Given that you're using this as a contains check later, consider turning it into a set: `demonyms = {entity.strip() for entity in fp}`

Reply:
fixed!